config/routes.js

```diff
@@ -10,6 +10,9 @@ module.exports = function(app, passport, auth) {
     app.post('/api/auth/signup', users.register);
     app.post('/api/auth/signin', users.login);
 
+    // Search all current users
+    app.get('/api/search/users', users.authToken, users.searchUsers);
```

Comment:
work with Boswell to make sure that when there is a token, it is actually authenticated